### DISCUSSION TITLE
今天的leetcode 的网络好像有点问题啊

### DISCUSSION THREAD

**Discussion by CTCSU** [ccagml/leetcode-extension#197] [General]
login 的时候都timeout 好几次了：
`login: pass: {"code":-2,"msg":{"errno":-4039,"code":"ETIMEDOUT","syscall":"connect","address":"104.18.13.180","port":443}}`

**Comment by ccagml** [comment]
最近感觉好像 CloudFlare 用起来怪怪的,很不流畅

**Reply by CTCSU** [reply]
cloudflare我不了解
不过今天leetcode 网站的响应也很慢，有的时候报错，我想是服务器弹性伸缩可能出问题了。

一开始登录了几次登不进 还以为是插件版本的问题 不过后来登录几次就又登录进去了。只是有的时候提交代码，search problem还是会加载不出来。感觉单纯就是服务器的问题吧，昨天还好好好的

**Reply by ccagml** [reply]
104.18.13.180 这个ip查着就是cloudflare, 官网也有问题吗?我很少用官网了, 一直以为是我本地网络有问题
